src/components/filterPatterns/popover/index.tsx

```diff
@@ -111,12 +111,14 @@ export const PopoverFilter: FC<PopoverFilterProps> = ({
                     alignItems="center"
```

Comment:
can we remove it on the span above then?

Reply:
Nuuuuu we cannot because there we have the flex which center the icon and text

Reply:
no no i mean just the `overflow="hidden" textOverflow="ellipsis"` part

Reply:
Nuuuuu we cannot. Otherwise the ellipsis is not showed

Reply:
so, I can remove only the textOverflow="ellipsis" the rest need to be there for displaying the ellipsis. Maybe removing textOverflow="ellipsis" on level up will help us to not break it again.